business_requirement_etherpad/README.rst

```diff
@@ -0,0 +1,166 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+
+=============================
+Business Requirement Etherpad
+=============================
+
+
+Configuration
+=============
+
+Users
+-----
+
+* **Business Requirement User**: can create and confirm a BR
+* **Business Requirement Manager**: can create, confirm and approve a BR
+
+Alias and emails
+----------------
+You can setup an alias in Settings/Technical/Email/Alias in order to create 
+business requirements directly from email received at a specific address.
+
+.. figure:: ../business_requirement/static/img/bus_req_alias.png
+   :width: 600 px
+   :alt: Email Alias setup
+
+You can start conversation with the followers in the chatter area of the 
+BR like in tasks, issue or CRM leads.
+
+Sequences
+---------
+
+Default numbering format: BR123456.
+
+You can modify the default numbering sequence in Settings/Technical/Sequences&Identifier/Sequences.
+
+Search for Business Requirement sequence and alter it if necessary.
+
+Tags
+----
+
+You can create and assign tags for your business requirements in Business Requirements/Configuration/Bus. Req. Category
+
+.. figure:: ../business_requirement/static/img/bus_req_tags.png
+   :width: 600 px
+   :alt: Define Tags
+
+
+Master project
+--------------
+
+You can define a master project linked to the business requirement.
+
+Followers from the project will automatically be added to the business requirements.
+
+
+Usage
+=====
+Simple BR
+---------
+
+1. Define the header information
+
```

Comment:
I think in usage a simple screenshot with the field and some lorem ipsum text is good enough.
@darshan-serpent